stable/joomla/README.md

```diff
@@ -63,8 +63,8 @@ The following table lists the configurable parameters of the Joomla! chart and t
 | `externalDatabase.port`           | Port of the external database          | `3306`                                                    |
 | `externalDatabase.user`           | Existing username in the external db   | `bn_joomla`                                               |
 | `externalDatabase.password`       | Password for the above username        | `nil`                                                     |
-| `externalDatabase.database`       | Name of the existing databse           | `bitnami_joomla`                                          |
-| `mariadb.enabled`                 | Wheter to use or not the mariadb chart | `true`                                                    |
+| `externalDatabase.database`       | Name of the existing database          | `bitnami_joomla`                                          |
+| `mariadb.enabled`                 | Whether to the MariaDB chart           | `true`                                                    |
```

Comment:
"Whether to the MariaDB chart" sounds strange to me. I am missing a verb

Reply:
My bad ... fat-fingered the change.

Reply:
Updated the PR. Please take a look. @javsalgar